src/Symfony/Component/Config/DefaultConfigCacheFactory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config;
+
+/**
+ * Default implementation for ConfigCacheFactoryInterface
+ * that will create an instance of ConfigCache.
+ *
+ * @author Matthias Pigulla <mp@webfactory.de>
+ */
+class DefaultConfigCacheFactory implements ConfigCacheFactoryInterface
```

Comment:
I don't like the name "Default". None implementation can be seen as default, each implementation has unique behaviour and based on all these implementations, a project can decide the default behaviour.

In this case, what about just naming it "ConfigCacheFactory"?